Sources/AWSLambdaEvents/AppSync.swift

```diff
@@ -0,0 +1,110 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2017-2020 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+public struct AppSync {
+	public struct Request: Codable {
```

Comment:
```suggestion
	public struct Event: Decodable {
```
All other events are named Event instead of Request. Moreover the `request` which contains the `HTTPHeaders` is missing